{{cookiecutter.project_slug}}/pyproject.toml

```
@@ -23,6 +23,7 @@ gunicorn = "23.0.0"
 psycopg = "3.2.9"
 {%- endif %}
 python-decouple = "3.8"
+python-json-logger = "2.0.7"
```

The latest version is 3.3.0

```suggestion
python-json-logger = "3.3.0"
```